test/tests/neutronics/tallies/fission/openmc.i

```diff
@@ -0,0 +1,92 @@
+[Mesh]
```

Comment:
Thoughts on perhaps consolidating these three tests into a single one? Could be easier to maintain in the future, since I don't anticipate any different behavior b/w these scores which we would want to modularize.